polychrom_hoomd/forces.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+from hoomd import md
+
+
+def kg_func(theta, kappa):
+    """Kremer-Grest bending energy penalty function"""
+
+    V = kappa * (1. + np.cos(theta))
+    T = kappa * np.sin(theta)
+
+    return V, T
+    
+    
+def poly_func(r, rmin, rmax, epsilon):
```

Comment:
would it make sense to separate force funcs into a separate file from force appliers?

Reply:
also maybe add some sensible defaults here for rmin, rmax, epsilon?